docs/how-to/how-to-create-standard-terraform-config.md

```diff
@@ -72,7 +72,11 @@ In general, a module should consist of resources unique to the component for whi
 
 The `environments` directory contains the Terraform config for different use cases. The environments separate the infrastructure for different use cases like development, staging and production projects or multiple instances of the same application. 
 The name of directories modules defined in `modules` directories are called by the config defined in the `environments` directory. Resources and variables specific for environments are passed to the modules as variables values. 
-Environments may call modules from any locations, not only modules defined under the same parent directory. That way, any module existing in `test-infra` and outside of it, can be reused. It's perfectly fine to have a Terraform config with `environments` directory only. Such config simply uses modules defined in other locations and provides the definition of resources specific for use case. Outputs returned by environments are published to the Terraform remote state. It's recommended to output all resources from an environment, so that other environments can consume it and use it as a dependency. 
+Environments may call modules from any locations, not only modules defined under the same parent directory. That way, any module existing in `test-infra` and outside of it, can be reused. It's perfectly fine to have a Terraform config with `environments` directory only. Such config simply uses modules defined in other locations and provides the definition of resources specific for use case. Outputs returned by environments are published to the Terraform remote state. It's recommended to output all resources from an environment, so that other environments can consume it and use it as a dependency.
```

Comment:
```suggestion
The environments may call modules from any locations, not only the modules defined under the same parent directory. That way, any module existing in `test-infra` and outside of it can be reused. Having a Terraform config with the `environments` directory is perfectly fine. Such a config simply uses the modules defined in other locations and provides the definition of resources specific to the use case. Outputs returned by the environments are published to the Terraform remote state. It's recommended to output all resources from an environment so that other environments can consume it and use it as a dependency.
```